backend/tests/test_contact.py

```diff
@@ -56,3 +56,47 @@ def test_delete_a_contact(self):
         # deleting a contact that does not exist
         response = self.client.delete("/contact/{}".format(c_id))
         self.assertEqual(response.status_code, 404)
+        
+    def test_edit_a_contact(self):
+        dummy_ct_id = uuid.uuid4()
+        dummy_ct = ContactType(
+            id=dummy_ct_id,
+            hex_colour="#ffffff",
+            type="dummy type",
+            description="dummy description",
+        )
+
+        c_id = uuid.uuid4()
+        c = Contact(
+            id=c_id,
+            name="dummy name",
+            email="dummy email",
+            secondary_email="dummy secondary email",
+            cellphone="dummy cellphone",
+            role="dummy role",
+            organization="dummy organization",
+            neighbourhood="dummy neighbourhood",
+            contact_type=dummy_ct,
```

Comment:
@dgiltz The reason why your tests are failing, is because at this line you are assigning a `ContactType` rather than the `uuid` of the `ContactType`. However, I think even if you were to assign the `uuid` of the `ContactType`, it would still fail. The reason is you probably want to assign `dummy_ct` to `c.contact_types` instead, this is due to the `backref` defined within the `ContactType` model.

Links:
https://github.com/hack4impact-mcgill/mu-crm-tool/blob/main/backend/app/models.py#L159
https://flask-sqlalchemy.palletsprojects.com/en/2.x/models/#one-to-many-relationships